storagenode/piecestore/endpoint.go

```diff
@@ -60,6 +60,7 @@ type Config struct {
 	DeleteQueueSize         int           `help:"size of the piece delete queue" default:"10000"`
 	OrderLimitGracePeriod   time.Duration `help:"how long after OrderLimit creation date are OrderLimits no longer accepted" default:"1h0m0s"`
 	CacheSyncInterval       time.Duration `help:"how often the space used cache is synced to persistent storage" releaseDefault:"1h0m0s" devDefault:"0h1m0s"`
+	PieceScanOnStart				bool          `help:"ifset to true, all pieces disk usage is recalculated on startup" default:"true"`
```

Comment:
Seems it’s not gofmt’ed. You could run `make goimports-fix` to properly format it.